exporter/elasticsearchexporter/elasticsearch_bulk.go

```diff
@@ -20,26 +20,25 @@ import (
 	"bytes"
 	"context"
 	"crypto/tls"
-	"errors"
 	"fmt"
 	"io"
-	"net"
 	"net/http"
 	"time"
 
 	"github.com/cenkalti/backoff/v4"
-	elasticsearch "github.com/elastic/go-elasticsearch/v8"
-	esutil "github.com/elastic/go-elasticsearch/v8/esutil"
```

Comment:
```suggestion
	elasticsearch "github.com/elastic/go-elasticsearch/v7"
	esutil "github.com/elastic/go-elasticsearch/v7/esutil"
```

Unless you're explicitly importing two seperate versions of the library, then the numbering isn't required.

Reply:
i am aware of that it needs to be version-specifically imported as the goal is also to have both versions depending on optional config for client version

Reply:
Just to clarify, you would like both version 7 and 8 to be part of this file?